File: JS, PHP, PYTHON/JAVASCRIPT/WHILE.js
const readline = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout
})


function tabla() {
    let continuar = 1;
    readline.question("Ingrese Ingrese un número para conocer la tabla de multiplicar:", input => {
        let num = parseInt(input);
        while (continuar === 1) {
            if (!isNaN(num)) {
                for (let i = 1; i <= 10; i++) {
                    console.log(`${num} x ${i} = ${num * i}`);
                }
                continuar = 0;
                readline.close();
                break;
                // Salir del bucle después de mostrar la tabla
            } else {
                console.log("Error, por favor ingrese un número entero");
            }
        }
    })
}

tabla();

File: JS, PHP, PYTHON/PYTHON/WHILE.py
def tabla():
    while True:
        try:
            num = int(input("Ingrese un numero para conocer la tabla de multiplicar: "))
            for i in range (1, 11):
                print(f"{num} x {i} = {num * i}")
        except:
            print("Error, por favor ingrese un numero entero")
            
tabla()